NUModules/NUModule.j

```diff
@@ -1140,7 +1142,13 @@ NUModuleTabViewModeIcon                = 2;
     if (!_isProcessingPush)
         [self showLoading];
 
-    if ([[self class] automaticContextManagement])
+    if (_usesPagination && [_categories count])
+    {
+        _latestCategoryLoadedIndex = 0;
+        var context = [_contextRegistry valueForKey:[_categories[0] contextIdentifier]];
```

Comment:
I would say `[_categories[_latestCategoryLoadedIndex]]`.
I feel `_latestCategoryLoadedIndex` should be renamed